examples/TemplateGenerationIterative.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>Note, the output ends here with a <code>KeyboardInterrupt</code>.  In the future, we will want to save the version in the repository with the full output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/HASI/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>